src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/TypeMapLoader.cs

```diff
@@ -36,7 +36,7 @@ public static IEnumerable<TypeMapping> LoadMappings(ImmutableArray<AdditionalTex
                             continue;
                         }
 
-                        var components = trimmedLine.Split('\t');
+                        var components = trimmedLine.Split(new char[] { '\t', ' ' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
I ended up making it support tabs and spaces both because I was concerned about the scenario where an editor turns tabs into spaces or something like that. Type names shouldn't include spaces, so splitting on both of these should work. I remove empty entries in the split output so `A\t\t\t   \t\t\tB` will still only be a two-item array.